doc/rst/tools/mason/mason.rst

```diff
@@ -159,202 +194,364 @@ To try out different values at runtime, pass the values for ``number`` to ``maso
 
 
 Testing your Project
-====================
+~~~~~~~~~~~~~~~~~~~~
 
-Testing a Mason project is easy! Start by adding test files to the ``test/`` directory and specifying
-them in your Mason.toml as follows:
+Mason makes it easy to test projects. To demonstrate how easy it is, the following is
+and example of adding to ``MyProject`` and running it. The test is as follows:
 
-.. code-block:: text
+.. code-block:: chpl
 
-    [brick]
-    name = "MyPackage"
-    version = "0.1.0"
-    chplVersion = "1.16.0"
-    authors = ["Sam Partee <Sam@Partee.com>"]
-    tests = ["sampleTest.chpl"]
+   config const testParam: bool = true;
 
-    [dependencies]
-    curl = '1.0.0'
+   if testParam {
+     writeln("Test Passed!");
+   }
+   else {
+     exit(1);
+   }
 
+Our project structure will be as follows::
 
-When the ``mason test [options]`` command is invoked, mason will find and download the necessary dependencies
-for your project that you listed in your Mason.toml and compile them with your main module found in
-``src/``. For example, after listing ``sampleTest.chpl`` as a test, the project structure would be
-as follows::
-
-    MyPackage/
-      Mason.toml
-      Mason.lock
-      src/
-        MyPackage.chpl
-      test/
-        sampleTest.chpl
-      target/
-        debug/
-        (release/)
-           MyPackage
-
-To test the project, run ``mason test [options]``, which will update the lock file, compile the tests,
-and produce the binary within ``target/test/`` as follows::
-
-    MyPackage/
-      Mason.toml
-      Mason.lock
-      src/
-        MyPackage.chpl
-      test/
-        sampleTest.chpl
-      target/
-        test/
-          sampleTest
-        debug/
-        (release/)
-          MyPackage
+  MyProject
```

Comment:
May be easier to distinguish directories from files by adding `/` at the end of directories.